Source/WebCore/Modules/streams/ReadableStream.cpp

```diff
@@ -475,6 +475,11 @@ JSC::JSValue JSReadableStream::cancel(JSC::JSGlobalObject& globalObject, JSC::Ca
     RefPtr internalReadableStream = wrapped().internalReadableStream();
     if (!internalReadableStream) {
         return callPromiseFunction(globalObject, callFrame, [this](auto& globalObject, auto& callFrame, auto&& promise) {
+            if (protectedWrapped()->isLocked()) {
```

Comment:
I suggest we use a local variable since `protectedWrapped()` is called several times in the same scope:
```
RefPtr wrapped = this->wrapped();
if (wrapped->isLocked()) {
```